react-ui/src/components/LogInForm.js

```diff
@@ -34,73 +34,93 @@ class logInForm extends Component {
     });
   };
 
-
   login = event => {
     let email = this.state.loginParams.email;
     let password = this.state.loginParams.password;
-    var params = {email: email, password: password}
-    axios.get(`/public/login/?${queryString.stringify(params)}`)
-    .then(res => {
-      if(res.data!==""){
-        this.setState({
-          email : res.data.email,
-          userType: res.data.userType,
-          firstName: res.data.firstName,
-          personId: res.data.workerId || res.data.studentId,
-          accessToken: res.data.accessToken,
-          islogged: true
+    let validinput = true;
+    var params = { email: email, password: password }
+
+    if (email === "" && password === "") {
+      validinput = false
+      alert("Please enter email and password")
+      event.preventDefault();
+    } else if (email === "") {
+      validinput = false;
+      alert("Please enter an email")
+      event.preventDefault();
+    } else if (password === "") {
+      validinput = false;
+      alert("please enter an password")
+      event.preventDefault();
+    }
+
+    if (validinput === true) {
+      axios.get(`/public/login/?${queryString.stringify(params)}`)
+        .then(res => {
+          if (res.data !== "") {
```

Comment:
Hi Kyle,
I would recommend checking null rather than an empty string as it currently would render an empty page (screenshot attached below) if the credentials are invalid instead of alerting the user that they entered incorrect credentials. 
![image](https://user-images.githubusercontent.com/32720322/99133769-55155080-25e9-11eb-8a43-9acd07f2584a.png)

Reply:
code has been updated to ensure that this screen shouldn't show up